schema.graphql

```diff
@@ -28,29 +28,52 @@ type Nft @entity(immutable: false) {
   type: NftType!
   tokenId: BigInt!
   platform: String!
+
   currentOwner: Account!
   previousOwner: Account!
   originalOwner: Account!
+
   currentRental: Rental
   rentalHistory: [Rental!]! @derivedFrom(field: "nft")
+  currentRentalOffer: RentalOffer
+  rentalOfferHistory: [RentalOffer!]! @derivedFrom(field: "nft")
 }
 
 type Account @entity(immutable: false) {
   id: ID!
   nfts: [Nft!]! @derivedFrom(field: "originalOwner")
   borrowings: [Rental!]! @derivedFrom(field: "borrower")
   lendings: [Rental!]! @derivedFrom(field: "lender")
+  rentalOffers: [RentalOffer!]! @derivedFrom(field: "lender")
```

Comment:
I don't think we will need this field at this moment. We can always list the NFTs of an account, and list the offers there

Reply:
```suggestion

```